docs/END_USER_DOCUMENT.md

```diff
@@ -310,7 +310,24 @@ That can be configured as follows during API definition:
 int addAuthor(@Var(value = "author", ignore = "books") Author author);
 ```
 
-In the example above, when calling `addAuthor`, Mocca will not include the list of books as variable in the payload of the GraphQL message.
+In the example above, when calling `addAuthor`, Mocca will not include the list of books as variable in the payload of the GraphQL message. 
+For a slightly more complicated example, let's say you wanted to add a book along with the book's author but don't want to include the list
+of books written by the author. You would use the `ignore` property like this:
```

Comment:
Can we go more than one level down in the hierarchy?  Can we clarify that in the document here?